graphyte.py

```diff
@@ -25,6 +25,12 @@
 default_sender = None
 logger = logging.getLogger(__name__)
 
+def has_whitespace(value):
+    if type(value) == int:
```

Comment:
Exact type checks aren't very Pythonic ("if it walks like a duck and quacks like a duck"). Let's change this to `if not isinstance(value, str)` and drop the `if type(value) == str` test altogether.

Reply:
Nice tip!
Well, actually I'm casting everything to _unicode_ before calling _has_whitespace so these tests aren't really necessary anymore.